configs/peer/genesis.json

```diff
@@ -241,42 +254,20 @@
           }
         },
         {
-          "Register": {
-            "NewRole": {
-              "id": "CONFIG_AND_USER_METADATA_ACCESS",
-              "permissions": [
-                {
-                  "definition_id": "can_change_config_parameters",
-                  "params": {
-                    "account_id": {
-                      "AccountId": "alice@wonderland"
-                    }
-                  }
-                },
-                {
-                  "definition_id": "can_remove_key_value_in_user_metadata",
-                  "params": {
-                    "account_id": {
-                      "AccountId": "alice@wonderland"
-                    }
-                  }
-                },
-                {
-                  "definition_id": "can_set_key_value_in_user_metadata",
-                  "params": {
-                    "account_id": {
-                      "AccountId": "alice@wonderland"
-                    }
-                  }
-                }
-              ]
+          "Grant": {
+            "PermissionToken": {
+              "definition_id": "allowed_to_do_stuff",
```

Comment:
```suggestion
              "definition_id": "can_set_parameters",
```

it should be this?

Reply:
Actually there are both. We have `can_set_parameters` granting below. `allowed_to_do_stuff` is just an example for some users who will read our `genesis.json` as I understand this. It has to other meaning and can be easilty removed.

Reply:
Please keep it. It's exactly that, an example.